chapters/ch11.md

```diff
@@ -1,57 +1,61 @@
 
 
 **3. The metric is measuring something other than what the project needs to optimize.**
 ->
-
+**3. Phép đo không phù hợp với mục tiêu tối ưu của dự án.**
 
 Suppose that for your cat application, your metric is classification accuracy. This metric currently ranks classifier A as superior to classifier B. But suppose you try out both algorithms, and find classifier A is allowing occasional pornographic images to slip through. Even though classifier A is more accurate, the bad impression left by the occasional pornographic image means its performance is unacceptable. What do you do?
 ->
-
+Giả sử trong ứng dụng mèo, phép đo của bạn là độ chính xác phân loại. Phép đo này hiện tại xếp hạng bộ phân loại A tốt hơn bộ phân loại B. Tuy nhiên, giả sử bạn thử cả hai thuật toán, và nhận ra rằng bộ phân loại A thi thoảng chấp nhận những bức ảnh khiêu dâm. Ngay cả khi bộ phân loại A chính xác hơn, ấn tượng xấu gây ra bởi một vài bức ảnh khiểu dâm đồng nghĩa với việc hiệu quả của nó là không chấp nhận được. Bạn sẽ làm gì?
 
 Here, the metric is failing to identify the fact that Algorithm B is in fact better than Algorithm A for your product. So, you can no longer trust the metric to pick the best algorithm. It is time to change evaluation metrics. For example, you can change the metric to heavily penalize letting through pornographic images. I would strongly recommend picking a new metric and using the new metric to explicitly define a new goal for the team, rather than proceeding for too long without a trusted metric and reverting to manually choosing among classifiers.
 ->
-
+Ở đây, phép đo không thành công trong việc xác định được thực tế Thuật toán B tốt hơn Thuật toán A trong sản phẩm của bạn. Bởi vậy, bạn không thể đặt niềm tin vào phép đo này để chọn thuật toán tốt nhất. Đây là lúc phải thay đổi phép đo. Ví dụ, bạn có thể thay đổi phép đo sao cho nó "phạt" thật nặng nếu một thuật toán chấp nhận ảnh khiêu dâm. Tôi khuyên bạn chọn một phép đo mới và sử dụng phép đo này để định nghĩa lại mục tiêu rõ ràng cho nhóm, hơn là tiếp tục chọn ra một cách thủ công trong số các bộ phân loại khi không có một phép đo đáng tin.
 
 It is quite common to change dev/test sets or evaluation metrics during a project. Having an initial dev/test set and metric helps you iterate quickly. If you ever find that the dev/test sets or metric are no longer pointing your team in the right direction, it’s not a big deal! Just change them and make sure your team knows about the new direction.
 ->
+Việc thay đổi tập phát triển/kiểm tra hoặc phép đo trong một dự án là khá phổ biến. Có một tập phát triển/kiểm tra và phép đo ban đầu giúp bạn hoàn thành chu kỳ phát triển một cách nhanh chóng. Nếu bạn nhận ra rằng tập phát triển/kiểm tra hoặc phép đo không còn giúp nhóm đi đúng hướng, không sao cả! Chỉ cần thay chúng và đảm bảo nhóm đồng thuận về hướng đi mới.
 
```

Comment:
"Không sao cả" thì sao phải thay đổi chúng?
It's not the big deal !
-> Tệ thật !

Reply:
> "Không sao cả" thì sao phải thay đổi chúng?
> It's not the big deal !
> -> Tệ thật !

Mình tin bạn chưa bao giờ nghe người Mỹ nói "It's not the big deal!". Mình chắc chắn câu này không có nghĩa là "Tệ thật!". 

Khi bạn làm một điều gì đó sai, người khác vẫn có thể nói với bạn rằng "không sao cả", và sau đó là "just change", tức "chỉ cần ...".

Reply:
- thêm từ "thì" trong câu cuối bắt đầu bằng Nếu.
- câu cuối thì câu gốc sát nghĩa là "để cả nhóm biết về hướng đi mới"

Reply:
> thêm từ "thì" trong câu cuối bắt đầu bằng Nếu.

Tớ thấy từ "thì" trong trường hợp này hơi thừa, "thì không sao cả" nghe không được tự nhiên. Trong tiếng Anh từ "then" đồng nghĩa với từ "thì" cũng không được sử dụng ở đây.